semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -363,6 +359,10 @@ def __init__(self, custom_values, members, enum_type):
     def __str__(self):
         return self.enum_type
 
+    @staticmethod
+    def __is_valid_enum_value__(val):
```

Comment:
Please don't use double-underscores, they are meant for certain special methods.

Just use 
```suggestion
    def is_valid_enum_value(val):
```